### DISCUSSION TITLE
Can we use auto-disposable StreamProvider inside StateNotifier?

### DISCUSSION THREAD

**Discussion by mgolokhov** [rrousselGit/riverpod#1462] [Q&A] [answered]
flutter_riverpod 1.0.3

<img width="631" alt="Screenshot 2022-06-16 at 21 04 45" src="https://user-images.githubusercontent.com/294512/174137029-befabff2-7c63-42e9-b8fc-3c374d67dd8e.png">

**Comment by rrousselGit** [comment, marked as answer]
Change Ref -> AutoDisposeRef
